src/components/ImageGeneration/GenerationForm/GenerateFormView.tsx

```diff
@@ -112,13 +113,15 @@ export function GenerateFormView({
   };
   // #endregion
 
-  // const [baseModel, aspectRatio, steps, quantity] = form.watch([
-  //   'baseModel',
-  //   'aspectRatio',
-  //   'steps',
-  //   'quantity',
-  // ]);
-  // const totalCost = calculateGenerationBill({ baseModel, aspectRatio, steps, quantity });
+  const [baseModel, aspectRatio, steps, quantity] = form.watch([
```

Comment:
When possible, it's better to subscribe to form changes rather than use form.watch. I'm not worried about it right now, but subscribing to form changes will lead to fewer component rerenders.